app/views/reservations/earliest_move_possible.html.haml

```diff
@@ -5,9 +5,9 @@
 - if @earliest_possible
   .modal-body
     - if @formatted_dates[:start_date] == @formatted_dates[:end_date]
-      %p= t('reservations.moving_up.confirm_same_day', @formatted_dates)
+      %p= t('reservations.moving_up.confirm_same_day', **@formatted_dates)
```

Comment:
Is `@formatted_dates` a hash?

Reply:
Indeed it is. It's set in the controller

```
      @formatted_dates = {
        start_date: human_date(@earliest_possible.reserve_start_at),
        start_time: human_time(@earliest_possible.reserve_start_at),
        end_date: human_date(@earliest_possible.reserve_end_at),
        end_time: human_time(@earliest_possible.reserve_end_at),
      }
```